docs/PR_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request Guide
+
+## General development rules:
+
+- All the changes of the code base are implemented through pull requests (PR) on master or other branches
+- When all next release features are implemented in master and master is stable enough release branch is made from it
+- Only PR with bugfix should be done on release. If it’s necessary it’s permitted to cherry-pick such bugfix PR to master
```

Comment:
s/release/the release branch/
(or release branches)

Reply:
- Only PR with bugfix should be done on release branches. If it’s necessary it’s permitted to cherry-pick such bugfix PRs to master